src/views/Home.jsx

```diff
@@ -1,11 +1,16 @@
 import './Home.css';
+import { Navigate } from 'react-router-dom';
+
+export function Home({ createNewList, listToken }) {
+	const handleClick = () => {
+		createNewList();
+	};
 
-export function Home() {
 	return (
 		<div className="Home">
-			<p>
-				Hello from the home (<code>/</code>) page!
-			</p>
+			{listToken != null && <Navigate to={'/List'} replace={true} />}
```

Comment:
Excellent use of the `ReactRouterDom.Navigate` component!

```suggestion
			{listToken && <Navigate to={'/List'} replace={true} />}
```
I would suggest taking advantage of the [`truthy`](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) and [`falsy`](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values within boolean contexts. We are expecting a token to be valid to navigate to `/List`. But, what if we have an empty string (`''`) or an `undefined` value (which are all `falsy` values)? There are two options to deal with this:
1. We can prevent the navigation to the `List` view by checking if an invalid/empty/undefined value is being passed as the `listToken` or
1. Push off the validation checking the validity of `listToken` within the `List` component

I think `listToken` validation can be punted on, so taking advantage of checking for a "truthy" `listToken` will meet the needs for now.

Reply:
@jendevelops A lot of learning here, thanks for the suggestion! When I check for truthiness using `{listToken && <Navigate...`, I am getting some unexpected behavior. If I clear local storage, we still get the expected behavior, however, when I set the key equal to `null` or `false` or `undefined`, the app redirects to /list. Since these are all falsy values, shouldn't this prevent the navigation? 
Would it be more clean to validate the token in app.jsx and pass a variable with a boolean value to home? (if so a @TODO for later)

Reply:
@wheninseattle I wonder if this is a weird browser cache thing sometimes. I'll have to pull down the branch later today and take a look locally and get back to you.

Reply:
@jendevelops, actually I figured it out. I thought since I was leaving `null` unquoted, it was actually sitting the value to null. In reality, it was making it a string 'null'! My mistake!